grafana-plugin/e2e-tests/integrations/integrationsTable.test.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '../fixtures';
+import { createIntegration } from '../utils/integrations';
+
+test('Integrations table shows data in Connections and Direct Paging tabs', async ({ adminRolePage: { page } }) => {
+  //   // Create 1 direct paging integration and 2 other connections
+  const WEBHOOK_INTEGRATION_NAME = 'Webhook-1';
+  const ALERTMANAGER_INTEGRATION_NAME = 'Alertmanager-1';
+  const DIRECT_PAGING_INTEGRATION_NAME = 'Direct paging-1';
```

Comment:
Won't this fail if run multiple times? Shouldn't this use a randomizer?

Reply:
Yep, good idea, that will make development of e2e easier, let me do that :) Good catch

Reply:
@teodosii it's done now. For Direct Paging item I did that a bit differently because there must be no duplicates of Direct Paging integration assigned to a single team. So I'm checking in test if one has already been created and I'm making sure that we don't try to create a duplicate. At this point its name doesn't matter